MerlinAU.sh

```diff
@@ -2869,37 +2861,41 @@ _Toggle_FW_UpdateCheckSetting_()
    _WaitForEnterKey_ "$mainMenuReturnPromptStr"
 }
 
-##-------------------------------------##
-## Added by Martinski W. [2024-Jan-25] ##
-##-------------------------------------##
+##----------------------------------------##
+## Modified by Martinski W. [2024-Mar-16] ##
+##----------------------------------------##
 _EntwareServicesHandler_()
 {
    if [ $# -eq 0 ] || [ -z "$1" ] ; then return 1 ; fi
 
-   local fileCount  entwOPT_init  entwOPT_unslung  actionStr=""
-
-   entwOPT_init="/opt/etc/init.d"
-   entwOPT_unslung="${entwOPT_init}/rc.unslung"
-
-   if [ ! -x /opt/bin/opkg ] || [ ! -x "$entwOPT_unslung" ]
-   then return 0 ; fi  ## Entware is not found ##
-
-   fileCount="$(/usr/bin/find -L "$entwOPT_init" -name "S*" -exec ls -1 {} \; 2>/dev/null | /bin/grep -cE "${entwOPT_init}/S[0-9]+")"
-   [ "$fileCount" -eq 0 ] && return 0
+   local serviceCnt  entwOPT_init  entwOPT_unslung  actionStr=""  divAction=""
 
    case "$1" in
-       stop) actionStr="Stopping" ;;
-      start) actionStr="Restarting" ;;
+       stop) actionStr="Stopping" ; divAction="unmount" ;;
```

Comment:
Ahhh, I see what you did here.
Yeah fair enough, it is starting to get a bit repetitive, time to cleanup and condense down. Approved.